docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2'
+services:
+  app:
+    container_name: app
+    build: .
+    ports:
+      - "17080:80"
+    env_file:
+      - .env
+    depends_on:
+      - db
+    networks:
+      - mafiascum
+
+  db:
+    container_name: mariadb
+    image: mariadb:latest
```

Comment:
We might want to consider deciding on a specific mariadb version, instead of just using latest, so that builds are repeatable and consistent.

(Obviously, we should bump the version as updates come out, it's just that I like to stay away from latest as it's ambiguous, especially when looking back in time)